browser_logs.log

```diff
@@ -0,0 +1,16 @@
+%cDownload the React DevTools for a better development experience: https://fb.me/react-devtools
```

Comment:
we recently added [`browser_logs.log` to the .gitignore](https://github.com/stride-nyc/remote_retro/blob/master/.gitignore#L45), so i think this branch may have been cut off of master before that change got in. @nicholalexander Can you rebase and repush to see if this file gets removed from the diff?

Reply:
yes!  should'a seen this.